android/tangram/src/com/mapzen/tangram/MapController.java

```diff
@@ -90,6 +94,21 @@
         void onLabelPick(LabelPickResult labelPickResult, float positionX, float positionY);
     }
 
+    /**
+     * Interface for a callback to receive the picked {@link Marker}
+     * Triggered after a call of {@link #pickMarker(float, float)}
+     * Listener should be set with {@link #setMarkerPickListener(MarkerPickListener)}
+     */
+    public interface MarkerPickListener {
+        /**
+         * Receive the picked {@link Marker}
+         * @param marker The {@link Marker} the marker that has been selected
+         * @param positionX The horizontal screen coordinate of the tapped location
+         * @param positionY The vertical screen coordinate of the tapped location
+         */
+        void onMarkerPick(Marker marker, float positionX, float positionY);
```

Comment:
Is there any way to get the current `LngLat` that was used to set the marker position from this callback?

https://github.com/tangrams/tangram-es/blob/master/android/tangram/src/main/java/com/mapzen/tangram/Marker.java#L96